README.md

```diff
@@ -3,7 +3,7 @@
 ## Self Introduction
 My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
 
-![Photo of Yuwen](C:\Users\Xyw\Documents\GitHub\CFSS\hw01\LinkedIn-profile-smaller.jpg)
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
```

Comment:
The picture is pretty big, but it's a really good picture, so that's chill.

Reply:
Thank you! I will try making it smaller next time.